minifi-bootstrap/src/main/java/org/apache/nifi/minifi/bootstrap/RunMiNiFi.java

```diff
@@ -1197,7 +1197,14 @@ public void start() throws IOException, InterruptedException {
         this.changeListener = new MiNiFiConfigurationChangeListener(this, defaultLogger);
         this.periodicStatusReporters = initializePeriodicNotifiers();
         startPeriodicNotifiers();
-        this.changeCoordinator = initializeNotifier(this.changeListener);
+        try {
+            this.changeCoordinator = initializeNotifier(this.changeListener);
+        } catch (Exception e) {
```

Comment:
I don't like how this catches any error and swallows the stack trace. Yes in this instance we know what is wrong but we may not in other cases. I'd prefer a more fine-grain catch block here.